<launch>
  <arg name="map_matcher_config_path" default="$(find-pkg-share map_matcher)/config/map_matcher.params.yaml"/>

  <set_parameter name="use_sim_time" value="true"/>
  <node pkg="map_matcher" exec="map_matcher_node" name="map_matcher" output="screen">
    <remap from="points_raw" to="/os1_cloud_node/points"/>
    <!--remap from="points_raw" to="/sensing/lidar/left_upper/pointcloud_raw"/-->
    <remap from="imu_raw" to="/os1_cloud_node/imu"/>
    <!--remap from="imu_raw" to="/sensing/imu/tamagawa/imu_raw"/-->
    <param from="$(var map_matcher_config_path)"/>
  </node>

</launch>
